app/controllers/api/v1/cars_controller.rb

```diff
@@ -25,6 +25,16 @@ def add_car
     end
   end
 
+  # delete a car item from list
+  def destroy
+    car = Car.find(params[:id])
+    if car.destroy
+      render json: { message: 'Car deleted successfully' }, status: :ok
+    else
+      render json: { head: unprocessable_entity }
+    end
+  end
+
```

Comment:
Nice job implementing the delete action.

Reply:
thanks! 😃